src/taipy/core/_scheduler/_scheduler.py

```diff
@@ -132,7 +143,10 @@ def __execute_jobs(cls):
             cls._update_job_config()
         while not cls.jobs_to_run.empty() and cls._dispatcher._can_execute():
             with cls.lock:
-                job = cls.jobs_to_run.get()
+                try:
+                    job = cls.jobs_to_run.get()
+                except:  # In case between line 144 and 145 the last job of the queue has been removed.
```

Comment:
```suggestion
                except:  # In case between line 144 and 145 the last job of the queue has been removed.
```
To improve, please, do not line number in a comment, this will be outdated quickly.

Reply:
Hi, I agree. @jrobinAV I don't recall writing this comment. Do you happen to know where it originated from? I think it explaining why the try catch is needed and it's safe to remove it.